hs/src/Reach/Debugger.hs

```diff
@@ -59,11 +59,11 @@ type Frontend = M.Map String ([DLVal] -> DLVal)
 --   | ShowAction String -- print the variable described by this string
 
 data Action
-  = ActionTieBreak
-  | ActionNewAcc
-  | ActionNewPart
-  | ActionImitateFrontend
-  | ActionInteract [DLVal]
+  = Action_TieBreak
+  | Action_NewAcc
+  | Action_NewPart
+  | Action_ImitateFrontend
+  | Action_Interact [DLVal]
```

Comment:
Sorry for being so sucky. We also use really short prefixes, like `A`. See https://github.com/reach-sh/reach-lang/blob/master/hs/src/Reach/AST/DLBase.hs for what I mean